sdk/storage/storage-blob-changefeed/samples/README.md

```diff
@@ -0,0 +1,83 @@
+# Getting started with samples
```

Comment:
IIRC there's some header/metadata that you can add to the top of README and it will enable publishing the samples folder as a zip to the microsoft docs site. (example: https://raw.githubusercontent.com/Azure/azure-sdk-for-js/master/sdk/formrecognizer/ai-form-recognizer/samples/javascript/README.md)

Reply:
Could you provide a sample of the published zip so I can verify my changes?

Reply:
Good point, @jeremymeng!

@ljian3377  This is where they get hosted - https://docs.microsoft.com/en-us/samples/browse

You can navigate through the side nav bar to reach the specific service and language..
Example - https://docs.microsoft.com/en-us/samples/browse/?languages=typescript&expanded=azure&products=azure-storage

Storage Blob example - https://docs.microsoft.com/en-us/samples/azure/azure-sdk-for-js/storage-blob-typescript/
(With the "download zip" button - zip has `samples/typescript` folder as the contents).

The metadata is supposed to be added at `samples/javascript/README.md` and `samples/typescript/README.md`.
(You can check `storage-blob/samples/typescript/README.md` for reference.)

Zip along with the readme will be published to `docs.microsoft.com` only after the PR is merged.
Once the metadata is present in those readmes, any updates to the samples folders will trigger the publishing which would override if there is an existing page at `docs.microsoft.com`.

Reply:
Done in the change feed PR: https://github.com/Azure/azure-sdk-for-js/pull/10827/commits/3eea9e58bd4f7b3c525431fec128d4f64b37a8b5